### PR TITLE
Add namespaces to HttpClients

### LINKED ISSUE #3: Add namespaces to IHttpClients
Same as https://github.com/trakx/copper-api-client/issues/6

### PR DESCRIPTION
**Pull request by AymericNoel**
close #3

### CODE REVIEW

File: src/Trakx.Fireblocks.ApiClient/openApi3.yaml [hunk 1]
```diff
@@ -8,7 +8,6 @@ paths:
   #  VAULT
```
Comment (monsieurleberre) [inline src/Trakx.Fireblocks.ApiClient/openApi3.yaml:8]: not sure I want this to be mereged 🤔
Yes I remember now I had added that manually, but maybe forgot there was something overriding the file somewhere.

**Review by monsieurleberre — CHANGES_REQUESTED** [review]
let's split the clients again?

**Comment by AymericNoel** [comment]
Should be ok by now

File: src/Trakx.Fireblocks.ApiClient/openApi3.yaml [hunk 2]
```diff
@@ -496,6 +496,32 @@ paths:
                 $ref: '#/components/schemas/PublicKeyInformation'
         default:
           $ref: '#/components/responses/Error'
+  '/vault/assets':
```
Comment (monsieurleberre) [inline src/Trakx.Fireblocks.ApiClient/openApi3.yaml:499]: OK I see that is what you had to do manually in the end, that is a pain, we will see how it goes on the long term...

File: src/Trakx.Fireblocks.ApiClient/openApi3.yaml [hunk 3]
```diff
@@ -1304,25 +1330,24 @@ paths:
...
-            type: string
-            minimum: 1
-      responses:
-        '200':
-          description: Unfreeze response
-          content:
-            '*/*':
-              schema:
-                $ref: '#/components/schemas/UnfreezeTransactionResponse'
+  # '/transactions/{txId}/unfreeze':
```
Comment (monsieurleberre) [inline src/Trakx.Fireblocks.ApiClient/openApi3.yaml:1333]: and this one, why don't we want it?

**Review by monsieurleberre — APPROVED** [review]
cool thanks 😁👍

File: src/Trakx.Fireblocks.ApiClient/openApi3.yaml (hunk 3 above)
Comment (AymericNoel) [reply src/Trakx.Fireblocks.ApiClient/openApi3.yaml:1333]: I have just updated the OpenAPi from fireblocks.... so they are the ones who don't want it anymore 😅

**Pull request merged**